thrimbletrimmer/edit.html

```diff
@@ -18,6 +18,8 @@
 			content="345276493482-r84m2giavk10glnmqna0lbq8e1hdaus0.apps.googleusercontent.com"
 		/>
 		<script src="https://apis.google.com/js/platform.js?onload=onGLoad" async defer></script>
+		<link rel="stylesheet" href="https://unpkg.com/jcrop/dist/jcrop.css">
+		<script src="https://unpkg.com/jcrop"></script>
```

Comment:
Can we ship the script ourselves (like we do with Luxon and hls.js) instead of relying on this site?